src/vivarium_conic_lsff/components/fortification/intervention.py

```diff
@@ -70,6 +72,11 @@ def setup(self, builder: 'Builder'):
                 'vitamin_a_fortification.coverage_level',
                 self.vitamin_a_intervention.adjust_coverage_level
             )
+        elif scenario == project_globals.SCENARIOS.IRON:
```

Comment:
As we talked about, abstract this out to a map from a scenario to a list of nutrients that will be scaled up.